api/db/changelog/db.changelog-211-app-info.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="nsaxena" id="changelog-211-app-info">
+    <createTable tableName="user_workspace_app">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
+        <constraints
+            nullable="false"
+            foreignKeyName="fk_user_workspace_app_user"
+            references="user(user_id)"
+            deleteCascade="true"
+        />
+      </column>
+      <column name="workspace_id" type="bigint">
+        <constraints
+            nullable="false"
+            foreignKeyName="fk_user_workspace_app_workspacw"
+            references="workspace(workspace_id)"
+            deleteCascade="true"
+        />
+      </column>
+      <column name="app_name" type="varchar(100)">
+        <constraints
+            nullable="false"
+        />
+      </column>
+      <column name="app_type" type="ENUM('CROMWELL')">
```

Comment:
looks funny to see a single-value enum here ... but I agree with the choice because it's the only one we're sure we'll need 👍